packages/prosemirror-lwdita/src/github-integration/github.plugin.ts

```diff
@@ -113,13 +113,13 @@ export const getUserInfo = async (token: string): Promise<Record<string, string>
  * @param changedDocument - The content of the changed document.
  * @returns A promise that resolves when the document has been published.
  */
-export const createPrFromContribution = async (ghrepo: string, source: string, changedDocument: string, title: string, desc: string): Promise<string> => {
+export const createPrFromContribution = async (ghrepo: string, source: string, branch: string, changedDocument: string, title: string, desc: string): Promise<string> => {
```

Comment:
tsdoc params need updating.

Reply:
Fixed in b6962288f2f38f863a94da06d3e6f53b77fe42fa .